src/data_manager/sub_type_config.py

```diff
@@ -21,57 +26,78 @@ def __init__(self, data_type, sub_data_type, file_to_download, filepath):
         self.already_downloaded = False
 
     def get_filepath(self):
+        '''Get filepath'''
+
         return self.filepath
 
     def get_sub_data_type(self):
+        '''Get sub data type'''
+
         return self.sub_data_type
 
     def get_file_to_download(self):
+        '''Get file to download'''
+
         return self.file_to_download
 
     def get_data_provider(self):
+        '''Get data provider'''
+
         return self.sub_data_type
 
     def get_data(self):
+        '''get data'''
+
         # Grab the data (TODO validate).
         # Some of this algorithm is temporary.
         # e.g. Files from the submission system will arrive without the need for unzipping, etc.
-        path = 'tmp'
+        download_folder = 'tmp'
```

Comment:
we use download_folder and download_dir in this PR - should we sync up on one or the other?

Reply:
Yes, I think I prefer dowload_dir. I will make the change.